lnd_services.go

```diff
@@ -25,6 +25,10 @@ var (
 	// call to find out if lnd is fully synced to its chain backend.
 	chainSyncPollInterval = 5 * time.Second
 
+	// defaultUnlockedInterval is the default amount of time we wait between
+	// checks that the wallet is unlocked.
+	defaultUnlockedInterval = 2 * time.Second
```

Comment:
I've tried this twice on my machine. Once it worked and once it didn't (was doing some compilation work so the machine was slow), throwing a "connection refused" error. I think we should increase this to 5 seconds, just to give slower machines a bigger chance. The slightly longer delay until the client is ready should be worth not having to start the client again (for example LiT crashing with a connection refused error).

Reply:
Could also allow more than one conn refused after unlock, say 3? That way faster machines can exit faster?
Def agreed that it's worth the wait, the 2s is just a magic nr.

Reply:
Sure, retrying multiple times would also work, if you are okay with rewriting the logic again :sweat_smile: